src/components/common/StudentAbout/_StudentAbout.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../styles/mixins';
+
+.div.component-StudentAbout {
```

Comment:
i jesteś pewien, że to ci działa ? Usuń kropkę z przed diva ;)

Reply:
Tak. `div` powinien być bez kropki